pyatv/mrp/__init__.py

```diff
@@ -380,8 +393,16 @@ def power_state(self):
         await self.remote.select()
 
     async def update_power_state(self, message, _):
```

Comment:
You can simplify this a bit now since `power_state` will return current power state:
```python
async def update_power_state(self, message, _):
    old_state = self.power_state
    self.device_info = message

    if self.power_state != old_state:
        _LOGGER.debug('Power state changed from %s to %s', old_state, self.power_state)
        if self.listener:
            self.loop.call_soon(self.listener.powerstate_update, self._current_power_state, self._new_power_state)
```

Reply:
the `new_state` was missing.

My code looks like this:
```
    async def update_power_state(self, message, _):
        old_state = await self.power_state()
        new_state = self._get_power_state(message)
        self.device_info = message

        if new_state != old_state:
            _LOGGER.debug('Power state changed from %s to %s', old_state, new_state)
            if self.listener:
                self.loop.call_soon(self.listener.powerstate_update, old_state, new_state)
```